Src/Repro/nuget.config

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!-- chrsmith -->
+    <add key="Private Support Libraries"
+        value="C:\src\github.com\chrsmith\google-api-dotnet-client\NuPkgs\Support">
+    </add>
+    <add key="Private Generated Libraries"
+        value="C:\src\github.com\chrsmith\google-api-dotnet-client\NuPkgs\Generated">
+    </add>
+    <!--
+      We do NOT want to reach out to NuGet.org and get the official packages.
```

Comment:
the official packages have dependencies like log4net or json.net that need to come from NuGet. Keep this source enabled but leave it last, and NuGet should pick things up from here.